<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebPageForm</class>
 <widget class="QWidget" name="WebPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="KanmusuMemory.qrc">
    <normaloff>:/resources/ShiromusuMemory32.png</normaloff>:/resources/ShiromusuMemory32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="goBackButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="KanmusuMemory.qrc">
           <normaloff>:/resources/go_back.png</normaloff>:/resources/go_back.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goForwardButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="KanmusuMemory.qrc">
           <normaloff>:/resources/go_forward.png</normaloff>:/resources/go_forward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="KanmusuMemory.qrc">
           <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="favoriteButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="KanmusuMemory.qrc">
           <normaloff>:/resources/fav_icon_off.png</normaloff>
           <normalon>:/resources/fav_icon_on.png</normalon>:/resources/fav_icon_off.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeMobileModeButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="KanmusuMemory.qrc">
           <normaloff>:/resources/mobile_icon_off.png</normaloff>
           <normalon>:/resources/mobile_icon_on.png</normalon>:/resources/mobile_icon_off.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="urlEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="findLayout">
     <item>
      <widget class="QLineEdit" name="findEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WebView" name="webView" native="true">
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>webview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="KanmusuMemory.qrc"/>
 </resources>
 <connections/>
</ui>
